### ISSUE TITLE
[Bug] Ryujinx not connecting to lanplay

### ISSUE THREAD

**Issue by darkarrow** [Ryujinx/Ryujinx#6144]
### Description of the issue

ryujinx master build after version 1083 not connecting to lanplay. Was working with Berry in the discord https://discord.com/channels/410208534861447168/410208610455519243/1197341619209379960

Versions I tested,
1092 -> no work
1087 -> no work
1084 -> no work
1083 -> works
1082 -> works
1065 -> works

### Reproduction steps

Using ryujinx on windows PC with two NICs, in my case vNICs, and try setting one NIC for lanplay and having ryujinx use it for ldn_mitm multiplayer. Then try hosting a room on a game such as mario kart 8.

### Log file

<details>
<summary>Log file</summary>

```
00:00:00.039 |I| Configuration LogValueChange: ResScale set to: 1
00:00:00.046 |I| Configuration LogValueChange: ResScaleCustom set to: 1
00:00:00.046 |I| Configuration LogValueChange: MaxAnisotropy set to: -1
00:00:00.047 |I| Configuration LogValueChange: AspectRatio set to: Fixed16x9
00:00:00.048 |I| Configuration LogValueChange: BackendThreading set to: Auto
00:00:00.048 |I| Configuration LogValueChange: GraphicsBackend set to: Vulkan
00:00:00.049 |I| Configuration LogValueChange: PreferredGpu set to: 0x10DE_0x17C8
00:00:00.049 |I| Configuration LogValueChange: AntiAliasing set to: None
00:00:00.050 |I| Configuration LogValueChange: ScalingFilter set to: Bilinear
00:00:00.050 |I| Configuration LogValueChange: ScalingFilterLevel set to: 80
00:00:00.051 |I| Configuration LogValueChange: EnableDockedMode set to: True
00:00:00.056 |I| Configuration LogValueChange: EnableVsync set to: True
00:00:00.056 |I| Configuration LogValueChange: EnableShaderCache set to: True
00:00:00.056 |I| Configuration LogValueChange: EnableTextureRecompression set to: False
00:00:00.057 |I| Configuration LogValueChange: EnableMacroHLE set to: True
00:00:00.057 |I| Configuration LogValueChange: EnableColorSpacePassthrough set to: False
00:00:00.057 |I| Configuration LogValueChange: EnablePtc set to: True
00:00:00.057 |I| Configuration LogValueChange: EnableInternetAccess set to: True
00:00:00.057 |I| Configuration LogValueChange: EnableFsIntegrityChecks set to: True
00:00:00.057 |I| Configuration LogValueChange: FsGlobalAccessLogMode set to: 0
00:00:00.057 |I| Configuration LogValueChange: AudioBackend set to: SDL2
00:00:00.057 |I| Configuration LogValueChange: AudioVolume set to: 1
00:00:00.059 |I| Configuration LogValueChange: MemoryManagerMode set to: HostMappedUnsafe
00:00:00.059 |I| Configuration LogValueChange: ExpandRam set to: False
00:00:00.059 |I| Configuration LogValueChange: IgnoreMissingServices set to: False
00:00:00.059 |I| Configuration LogValueChange: UseHypervisor set to: True
00:00:00.061 |I| Configuration LogValueChange: MultiplayerMode set to: LdnMitm
00:00:00.061 |N| Application PrintSystemInfo: Ryujinx Version: 1.1.1084
00:00:00.065 |N| Application Print: Operating System: Microsoft Windows 10.0.22000 (X64)
00:00:00.065 |N| Application Print: CPU: AMD Ryzen 7 5800X 8-Core Processor ; 16 logical
00:00:00.066 |N| Application Print: RAM: Total 31.9 GiB ; Available 20.3 GiB
00:00:00.068 |N| Application PrintSystemInfo: Logs Enabled: Info, Warning, Error, Guest, Stub
00:00:00.068 |N| Application PrintSystemInfo: Launch Mode: UserProfile
00:00:00.064 |I| Gpu : Backend Threading (Auto): True
00:00:00.232 |N| Application LoadApplication: Using Firmware Version: 17.0.1
00:00:00.233 |I| Application LoadApplication: Loading as NSP.
00:00:00.254 |I| Application AddAocItem: Found AddOnContent with TitleId 0100152000023001
00:00:00.267 |I| ModLoader QueryContentsDir: Searching mods for Title 0100152000022000
00:00:00.267 |I| ModLoader QueryContentsDir: Searching mods for DLC 0100152000023001
00:00:00.267 |I| ModLoader QueryContentsDir: Searching mods for Title 0100152000022000
00:00:00.268 |I| ModLoader QueryContentsDir: Searching mods for DLC 0100152000023001
00:00:00.269 |I| Loader Load: Loading rtld...
00:00:00.276 |I| Loader PrintRoSectionInfo: rtld:
    Module: nnrtld
00:00:00.276 |I| Loader Load: Loading main...
00:00:00.334 |I| Loader PrintRoSectionInfo: main:
    Module: D:\home\Project\TurboS_chg6\App\Rom\NX32\Product\TurboS.nss
    SDK Libraries: SDK MW+Nintendo+PiaCommon-5_6_4
                   SDK MW+Nintendo+Pia-5_6_4
                   SDK MW+Nintendo+NEX-4_3_3-apptrbs
                   SDK MW+Nintendo+NintendoSDK_gfx-4_4_0-Release
                   SDK MW+Nintendo+NintendoWare_G3d-4_4_0-Release
                   SDK MW+Nintendo+NintendoWare_Vfx-4_4_0-Release
                   SDK MW+Nintendo+NintendoWare_Atk-4_4_0-Release
                   SDK MW+Nintendo+NEX_CO-4_3_3
                   SDK MW+Nintendo+PiaInet-5_6_4
                   SDK MW+Nintendo+PiaNat-5_6_4
                   SDK MW+Nintendo+PiaLan-5_6_4
                   SDK MW+Nintendo+PiaSession-5_6_4
                   SDK MW+Nintendo+PiaTransport-5_6_4
                   SDK MW+Nintendo+PiaLocal-5_6_4
                   SDK MW+Nintendo+NintendoSDK_libcurl-4_4_0-Release
                   SDK MW+Nintendo+NintendoWare_Font-4_4_0-Release
                   SDK MW+Nintendo+NintendoWare_Ui2d-4_4_0-Release
                   SDK MW+Nintendo+NEX_MM-4_3_3
                   SDK MW+Nintendo+NEX_DS-4_3_3
                   SDK MW+Nintendo+NEX_RK-4_3_3
                   SDK MW+Nintendo+NEX_UT-4_3_3
                   SDK MW+Nintendo+NintendoSDK_libz-4_4_0-Release
00:00:00.335 |I| Loader Load: Loading subsdk0...
00:00:00.352 |I| Loader PrintRoSectionInfo: subsdk0:
    Module: multimedia
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_movie-4_4_0-Release
00:00:00.352 |I| Loader Load: Loading sdk...
00:00:00.379 |I| Loader PrintRoSectionInfo: sdk:
    Module: nnSdk
    FS SDK Version: 4.4.0
    SDK Libraries: SDK MW+Nintendo+NintendoSDK_libz-4_4_0-Release
                   SDK MW+Nintendo+NintendoSdk_nnSdk-4_4_0-Release
                   SDK MW+Nintendo+NintendoSDK_NVN-4_4_0-Release
00:00:00.465 |I| Ptc Initialize: Initializing Profiled Persistent Translation Cache (enabled: True).
00:00:00.816 |I| Ptc Load: Loaded Translation Cache (size: 36919135 bytes, translated functions: 52673).
00:00:00.829 |I| Ptc Load: Loaded Profiling Info (size: 150146 bytes, profiled functions: 50498).
00:00:00.835 |I| Loader LoadNsos: Loading image 0 at 0x0000000000700000...
00:00:00.836 |I| Loader LoadNsos: Loading image 1 at 0x0000000000704000...
00:00:00.840 |I| Loader LoadNsos: Loading image 2 at 0x0000000001660000...
00:00:00.842 |I| Loader LoadNsos: Loading image 3 at 0x0000000001c2e000...
00:00:00.847 |I| ModLoader LoadCheats: Build ids found for title 0100152000022000:
    392D5F620063DCF173BD5C4EA2DE7D46733A7184000000000000000000000000
    9EF5CAA2D5B933C772358C5AA6FABA1500000000000000000000000000000000
    EA3F2A9B705EEF46ED08414F88C49ECF2559B4FB000000000000000000000000
    F0D9307B48E4E8F147A40E44A9CBC555D4C6148F000000000000000000000000
00:00:00.849 |I| Application EnsureSaveData: Ensuring required savedata exists.
00:00:00.890 |I| Loader Start: Application Loaded: Mario Kart 8 Deluxe v3.0.1 [0100152000022000] [32-bit]
00:00:00.896 |I| GUI.WindowThread Hid Configure: Configured Controller JoyconPair to Player1
00:00:00.906 |I| GUI.WindowThread Hid SetupNpad: Connected Controller JoyconPair to Player1
00:00:01.180 |N| GUI.RenderLoop Gpu PrintGpuInformation: NVIDIA NVIDIA GeForce GTX 980 Ti (Vulkan v1.3.260, Driver v546.1.0.0)
00:00:01.186 |I| GPU.MainThread Gpu LoadShaders: Loading 253 shaders from the cache...
00:00:01.570 |I| GPU.MainThread Gpu LoadShaders: Shader cache loaded.
00:00:02.439 |I| HLE.GuestThread.42 Ptc LoadTranslations: 50498 translated functions loaded
00:00:02.504 |S| HLE.OsThread.43 ServiceAm GetAppletResourceUserId: Stubbed. {appletResourceUserId: 1}
00:00:02.507 |S| HLE.OsThread.43 ServiceAm AcquireForegroundRights: Stubbed. 
00:00:02.507 |S| HLE.OsThread.43 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: False ; unknownFlag2: False ; unknownFlag3: True}
00:00:02.508 |S| HLE.OsThread.43 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:02.513 |S| HLE.OsThread.43 ServicePctl Initialize: Stubbed. 
00:00:03.153 |S| HLE.OsThread.10 ServiceNv FinishInitialize: Stubbed. 
00:00:03.158 |I| HLE.OsThread.43 SurfaceFlinger CreateLayerFromId: Creating layer 1
00:00:03.174 |S| HLE.OsThread.10 ServiceNv InitializeEx: Stubbed. 
00:00:03.183 |S| HLE.OsThread.10 ServiceNv GetActiveSlotMask: Stubbed. 
00:00:03.215 |S| HLE.OsThread.10 ServiceNv SetNvMapFd: Stubbed. 
00:00:03.216 |S| HLE.OsThread.10 ServiceNv AllocGpfifoEx2: Stubbed. 
00:00:03.216 |S| HLE.OsThread.10 ServiceNv AllocObjCtx: Stubbed. 
00:00:03.217 |S| HLE.OsThread.10 ServiceNv SetTimeout: Stubbed. 
00:00:03.217 |S| HLE.OsThread.10 ServiceNv SetErrorNotifier: Stubbed. 
00:00:03.218 |S| HLE.OsThread.10 ServiceNv SetUserData: Stubbed. 
00:00:03.218 |S| HLE.OsThread.10 ServiceNv SetPriority: Stubbed. 
00:00:03.219 |S| HLE.OsThread.10 ServiceNv ZcullBind: Stubbed. 
00:00:03.246 |S| HLE.OsThread.9 ServiceHid ActivateTouchScreen: Stubbed. {appletResourceUserId: 1}
00:00:03.246 |S| HLE.OsThread.9 ServiceHid ActiveNpadImpl: Stubbed. {appletResourceUserId: 1 ; revision: 0}
00:00:03.249 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadIdType: Stubbed. 9 Players: Player1,Player2,Player3,Player4,Player5,Player6,Player7,Player8,Handheld
00:00:03.249 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadStyleSet: Stubbed. {pid: 119 ; appletResourceUserId: 1 ; type: ProController, Handheld, JoyconPair, JoyconLeft, JoyconRight}
00:00:03.250 |S| HLE.OsThread.9 ServiceHid SetSupportedNpadIdType: Stubbed. 5 Players: Player1,Player2,Player3,Player4,Handheld
00:00:03.365 |S| HLE.OsThread.43 ServiceAcc InitializeApplicationInfo: Stubbed. {TitleId: 72080821221203968}
00:00:03.400 |S| HLE.OsThread.9 ServiceHid SetNpadHandheldActivationMode: Stubbed. {appletResourceUserId: 1 ; _npadHandheldActivationMode: Dual}
00:00:03.507 |S| HLE.OsThread.43 ServiceAcc CheckAvailability: Stubbed. 
00:00:03.507 |S| HLE.OsThread.43 ServiceAcc GetNintendoAccountUserResourceCacheForApplication: Stubbed. {NetworkServiceAccountId: 51966}
00:00:03.507 |S| HLE.OsThread.43 ServiceAm LockExit: Stubbed. 
00:00:03.514 |S| HLE.OsThread.43 ServiceAm UnlockExit: Stubbed. 
00:00:03.519 |S| HLE.OsThread.43 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: False ; unknownFlag3: True}
00:00:03.520 |S| HLE.OsThread.43 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:03.520 |S| HLE.OsThread.43 ServiceAm SetOperationModeChangedNotification: Stubbed. {operationModeChangedNotification: True}
00:00:03.520 |S| HLE.OsThread.43 ServiceAm SetPerformanceModeChangedNotification: Stubbed. {performanceModeChangedNotification: True}
00:00:03.520 |S| HLE.OsThread.43 ServiceAm SetRestartMessageEnabled: Stubbed. {restartMessageEnabled: True}
00:00:04.142 |I| HLE.OsThread.6 AudioRenderer AcquireSessionId: Registered new renderer (0)
00:00:04.143 |I| HLE.OsThread.6 AudioRenderer Initialize: Initializing with REV4
00:00:04.155 |I| AudioProcessor.Worker AudioRenderer Work: Starting audio processor
00:00:04.156 |I| AudioRendererManager.Worker AudioRenderer SendCommands: Starting audio renderer
00:00:04.184 |I| HLE.OsThread.6 AudioRenderer Start: Starting renderer id 0
00:00:04.201 |I| AudioProcessor.Worker Audio EnsureAudioStreamSetup: New audio stream setup with a target sample count of 240
00:00:06.608 |S| HLE.OsThread.9 ServiceHid StartSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 5}
00:00:06.609 |S| HLE.OsThread.9 ServiceHid StartSixAxisSensor: Stubbed. {appletResourceUserId: 1 ; sixAxisSensorHandle: 65541}
00:00:06.613 |S| HLE.OsThread.43 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 1}
00:00:06.614 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=0100152000023001
00:00:06.614 |S| HLE.OsThread.43 ServiceNs PrepareAddOnContentImpl: Stubbed. {index: 1}
00:00:06.615 |I| HLE.OsThread.8 Loader OpenDataStorageByDataId: Opened AddOnContent Data TitleID=0100152000023001
00:00:06.664 |I| HLE.OsThread.24 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 119
 UserId: 815021554af04580be14c0066c49277e
 Room: dlc 
 Report: {
    "CheckedTime": 20240118003102,
    "DlcState": 3,
    "DlcVersion": 10002
}

00:00:06.710 |S| HLE.OsThread.43 ServiceAm LockExit: Stubbed. 
00:00:06.726 |S| HLE.OsThread.43 ServiceAm UnlockExit: Stubbed. 
00:00:06.733 |I| HLE.OsThread.24 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 119
 UserId: 815021554af04580be14c0066c49277e
 Room: user_data 
 Report: {
    "Version": "3.0.1",
    "StartupCount": 34,
    "GrandPrixTrophy": "000000000000000000000000000000000000000000000000000000000000",
    "GrandPrixTrophyCnsw": "000000000000000000000000000000000000000000000000000000000000",
    "TotalRaceNum": 0,
    "TotalBalloonBattleNum": 0,
    "TotalKeidoroBattleNum": 0,
    "TotalBombBattleNum": 0,
    "TotalCoinBattleNum": 0,
    "TotalShineBattleNum": 0,
    "TotalMileageMeter": 0,
    "TotalRunTimeFrame": 0,
    "TotalDrift": 0,
    "TotalMiniTurbo1": 0,
    "TotalMiniTurbo2": 0,
    "TotalMiniTurbo3": 0,
    "TotalJumpAction": 0,
    "TotalJugemRescue": 0,
    "TotalGyroTimeFrame": 0,
    "TotalAssistTimeFrame": 0,
    "TotalAutoAccelTimeFrame": 0,
    "TotalShowMapTimeFrame": 0,
    "TotalFlyTimeFrame": 0,
    "TotalSwimTimeFrame": 0,
    "TotalCoin": 0,
    "TotalDisconnectCount": 0,
    "TotalWin": 0,
    "TotalLose": 0,
    "RaceRate": 1000,
    "BattleRate": 1000,
    "OpenGoldMario": 0,
    "OpenBodyFlags": {
        "TypeCode": 0,
        "Value": "0x0000014A0080C829"
    },
    "OpenTireFlags": {
        "TypeCode": 0,
        "Value": "0x000000000000C829"
    },
    "OpenWingFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000004829"
    },
    "Win150ccStaffGhostFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Win150ccStaffGhostCount": 0,
    "Try150ccTimeAttackFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Try150ccTimeAttackCount": 0,
    "Win150ccStaffGhostFlagsCnsw": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Win150ccStaffGhostCountCnsw": 0,
    "Try150ccTimeAttackFlagsCnsw": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Try150ccTimeAttackCountCnsw": 0,
    "Win200ccStaffGhostFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Win200ccStaffGhostCount": 0,
    "Try200ccTimeAttackFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Try200ccTimeAttackCount": 0,
    "Win200ccStaffGhostFlagsCnsw": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Win200ccStaffGhostCountCnsw": 0,
    "Try200ccTimeAttackFlagsCnsw": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "Try200ccTimeAttackCountCnsw": 0,
    "WatchEndingFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "OpenMiiSuitsFlags": {
        "TypeCode": 0,
        "Value": "0x0000000000000000"
    },
    "DlcState": 3
}

00:00:13.346 |S| HLE.OsThread.43 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: False ; unknownFlag3: True}
00:00:13.347 |S| HLE.OsThread.43 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:15.130 |I| HLE.OsThread.24 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 119
 UserId: 815021554af04580be14c0066c49277e
 Room: ui 
 Report: {
    "SequenceTime": 20240118003111,
    "Version": "3.0.1",
    "SequenceNumber": 1,
    "To": "Local-Single",
    "From": "TitleSelect"
}

00:00:15.130 |S| HLE.OsThread.43 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: True ; unknownFlag3: False}
00:00:15.130 |S| HLE.OsThread.43 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:15.167 |S| HLE.OsThread.5 ServiceBsd RegisterClient: Stubbed. 
00:00:15.167 |S| HLE.OsThread.5 ServiceBsd StartMonitoring: Stubbed. {unknown0: 0}
00:00:16.140 |I| HLE.OsThread.15 ServiceLdn : Initializing with multiplayer mode: LdnMitm
00:00:16.146 |S| HLE.OsThread.43 ServiceAm GetPseudoDeviceId: Stubbed. 
00:00:16.162 |I| HLE.OsThread.15 ServiceLdn : Initialize LanDiscovery using IP: 10.13.238.62
00:00:16.167 |S| HLE.OsThread.15 ServiceLdn : LdnMitmClient SetGameVersion
00:00:16.597 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player5}
00:00:16.597 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player6}
00:00:16.597 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player7}
00:00:16.597 |S| HLE.OsThread.9 ServiceHid DisconnectNpad: Stubbed. {appletResourceUserId: 4294967296 ; npadIdType: Player8}
00:00:17.187 |I| HLE.OsThread.15 ServiceLdn : LdnProxyTCPServer created a server for this address: 10.13.238.62:11452
00:00:17.196 |S| HLE.OsThread.43 ServiceAm GetPseudoDeviceId: Stubbed. 
00:00:17.216 |I| HLE.OsThread.15 ServiceLdn GetIpv4Address: Console's LDN IP is "10.13.238.62".
00:00:17.246 |I| HLE.OsThread.15 ServiceLdn GetIpv4Address: Console's LDN IP is "10.13.238.62".
00:00:17.275 |I| HLE.OsThread.15 ServiceLdn GetIpv4Address: Console's LDN IP is "10.13.238.62".
00:00:17.345 |S| HLE.OsThread.43 ServiceAm GetPseudoDeviceId: Stubbed. 
00:00:17.384 |I| HLE.OsThread.15 ServiceLdn GetIpv4Address: Console's LDN IP is "10.13.238.62".
00:00:32.461 |I| .NET TP Worker Ptc Save: Saved Profiling Info (size: 150427 bytes, profiled functions: 50611).
00:00:39.329 |S| HLE.OsThread.43 ServiceAm GetPseudoDeviceId: Stubbed. 
00:00:39.480 |I| HLE.OsThread.24 ServicePrepo ProcessPlayReport: 
PlayReport log:
 Kind: Normal
 Pid: 119
 UserId: 815021554af04580be14c0066c49277e
 Room: session 
 Report: {
    "Version": "3.0.1",
    "BeginTime": 20240118003113,
    "Mode": "Local",
    "LobbyCode": "",
    "GatheringID": 0,
    "EndTime": 20240118003135
}

00:00:40.530 |S| HLE.OsThread.43 ServiceAm SetFocusHandlingMode: Stubbed. {unknownFlag1: True ; unknownFlag2: False ; unknownFlag3: True}
00:00:40.530 |S| HLE.OsThread.43 ServiceAm SetOutOfFocusSuspendingEnabled: Stubbed. {outOfFocusSuspendingEnabled: False}
00:00:45.988 |W| HLE.GuestThread.44 KernelSvc : SendSyncRequest() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.42 KernelSvc : ArbitrateLock() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.49 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.45 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.50 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.46 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.53 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.52 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.48 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.55 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.67 KernelSvc : WaitForAddress() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.56 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.51 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.79 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.57 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.54 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.58 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.80 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.90 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.59 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.91 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.60 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.69 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.70 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.988 |W| HLE.GuestThread.61 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.71 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.62 KernelSvc : WaitSynchronization(handleIndex: 0xFFFFFFFF) = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.72 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.68 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.76 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.73 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.77 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.75 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.78 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.84 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.85 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.86 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.87 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.88 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.989 |W| HLE.GuestThread.89 KernelSvc : WaitProcessWideKeyAtomic() = TerminationRequested
00:00:45.994 |E| HLE.OsThread.13 SurfaceFlinger DequeueBuffer: No available buffer slots
00:00:45.995 |E| HLE.OsThread.13 SurfaceFlinger OnTransact: Error returned by transaction DequeueBuffer: Busy
00:00:46.002 |I| HLE.OsThread.6 AudioRenderer Stop: Stopping renderer id 0
00:00:46.002 |I| HLE.OsThread.6 AudioRenderer Stop: Stopped renderer id 0
00:00:46.002 |I| HLE.OsThread.6 AudioRenderer ReleaseSessionId: Unregistered renderer (0)
00:00:46.004 |I| AudioProcessor.Worker AudioRenderer Work: Stopping audio processor
00:00:46.006 |W| GUI.WindowThread ServiceNv Close: Closing channel
00:00:46.031 |I| HLE.OsThread.6 AudioRenderer StopLocked: Stopped audio renderer
```
</details> 

### OS

Windows 11

### Ryujinx version

1.1.1084+

### Game version

3.0.1

### List of applied mods

N/A

**Comment by TSRBerry** [comment]
@darkarrow Does version 1.1.1140 fix this by any chance?

**Comment by TSRBerry** [comment]
I tested this locally (not with lan-play directly) for a bit and don't run into the problems from before anymore.
I'd say this was probably fixed with 1.1.1140, but if there are still issues feel free to reopen this.
